plugins/link.md

```diff
@@ -233,7 +233,7 @@ tinymce.init({
 
 ### `rel_list`
 
-This option lets you specify a list of `rel` values for the `link` dialog. These values gets applied to the `rel` attribute. Each `rel` item must be defined as an object with a `title` and a `value`. For example: `{title: 'Table of contents', value: 'toc'}`. When the dialog is submitted, the `value` of the selected `rel` item will be set as the link's `rel` attribute.
+This option lets you specify a list of `rel` values for the `link` dialog. These values gets applied to the `rel` attribute. Each `rel` item must be defined as an object with a `title` and a `value`. For example: `{title: 'No Referral', value: 'noreferrer'}`. When the dialog is submitted, the `value` of the selected `rel` item will be set as the link's `rel` attribute.
```

Comment:
Should this be `No Referrer`? As `Referral` feels a bit out of place here, but maybe that's just me.

Reply:
Updated by 882010c6da6e884175c3ed9d81030571b4cb3160